mailpoet/assets/js/src/newsletters/listings/standard.jsx

```diff
@@ -187,6 +187,25 @@ class NewsletterListStandardComponent extends Component {
     };
   }
 
+  componentDidMount() {
+    // Send event that the MP wizard was loaded via the WooCommerce home page to Mixpanel and delete the corresponding setting.
+    // We need to do this after the user completes the wizard, since before that tracking is not enabled.
+    if (window.mailpoet_track_wizard_loaded_via_woocommerce) {
```

Comment:
I'm not sure if this is the best place to put the code that sends the Mixpanel event and updates the setting.

Reply:
Hmm. That is tricky. I'm currently working on the new homepage so after the homepage lands Emails will no longer be the page where we send users from the wizard.

I think we might not to put it anywhere in React, but maybe a simple `document.addEventListener('DOMContentLoaded', trackWizardLoaded)` could work. It can be placed into a separate file just needs to be added to [webpack_admin_index.tsx](https://github.com/mailpoet/mailpoet/blob/671c7eed7db526acb21e78cb8b3e6ff381e31685/mailpoet/assets/js/src/webpack_admin_index.tsx) so that it is bundled within admin JS which we include on every MailPoet admin page.
The DOMContentLoaded event should be enough to make sure `MailPoet.trackEvent` is available. We don't need to wait for Mixpanel to load because this is handled inside the `MailPoet.trackEvent` ([#](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/assets/js/src/analytics_event.js#L68-L75)).

What do you think about such an approach? Would that work for you?

Reply:
I went with the approach that you suggested.

The only downside that I see is that we are adding this script to every single page. That is not a big deal since it is the admin and it applies only to MP pages. I considered adding a check to load it only in the home page, but I thought this check could be a bit brittle and break in the future due to changes that we might implement. So I left the code as is.

Reply:
> The only downside that I see is that we are adding this script to every single page.

Good point! But in most cases, it only checks if `window.mailpoet_track_wizard_loaded_via_woocommerce` is set so I think it is not a big deal.

Reply:
Yeah, I agree